index.html

```diff
@@ -0,0 +1,136 @@
+                                <a href="#"
+                                    ><i class="item fab fa-apple"></i
+                                ></a>
+                            </li>
+                            <li class="item">
+                                <a href="#">Store</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">Mac</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">iPod</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">iPhone</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">iPad</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">iTunes</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">Support</a>
+                            </li>
+                            <li class="item">
+                                <div class="nav-input">
+                                    <i class="fas fa-search"></i
+                                    ><input class="input-search" type="text" />
+                                </div>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+                <header class="titles">
+                    <a href="#"><h1>What will your verse be?</h1></a>
+                    <a href="#"
+                        ><h2>
+                            Everyone has something to share. See how people are
+                            using<br />
+                            iPad in their unique ways to add to the world's
+                            story.
+                        </h2></a
+                    >
+                    <a href="#"
+                        ><h3>
+                            Watch the "Your Verse" film and explore the stories
+                        </h3></a
+                    ><a href="#"
+                        ><img
+                            class="play-btn"
+                            src="images/play-icon.png"
+                            alt="play button"
+                    /></a>
+                </header>
+            </main>
+            </div>
+            <div class="flex-container">
+                <div class="flex-item-div">
+                    <a href="#"
+                        ><img src="images/iPhone5s.jpg" alt="phone"
+                    /></a>
+                </div>
+                <div class="flex-item-div">
+                    <a href="#"
+                        ><img src="images/iPhone5c.jpg" alt="phone"
+                    /></a>
+                </div>
```

Comment:
- [ ] These two particular images are not being displayed. I noticed that is about a type. The files are saved as iphone5s and iphone5c, not iPhone5s and iPhone5c. Kindly fix that.

> screenshot 
![image](https://user-images.githubusercontent.com/57414789/82617074-8f4ee980-9b94-11ea-9bf1-43998412c393.png)